instrumentation/kafka/kafka-clients/kafka-clients-0.11/javaagent/src/test/java/io/lumigo/javaagent/instrumentation/kafkaclients/v0_11/KafkaClientTest.java

```diff
@@ -25,23 +25,23 @@
 import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
```

Comment:
The only reason I needed to change the original test is because we decide to not turn on KAFKA_TELEMETRY_SUPPORT. 
this cause a change in the traces structure so I needed to modify the tests.